src/app/containers/LoginPage/index.js

```diff
@@ -20,7 +20,7 @@ import { API_PROGRESS } from '../../../shared/constants';
 
 function LoginPage() {
   const [apiProgress, setApiProgress] = React.useState(API_PROGRESS.INIT);
-  const { userInfo } = React.useContext(UserInfoContext);
+  const { setUserInfo, userInfo } = React.useContext(UserInfoContext);
```

Comment:
@qtuan1401 please take note, `userInfo` only used to get the information not being used to set the information

To set the information in the context please use the `setUserInfo`